aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
+IIS Configuration
+^^^^^^^^^^^^^^^^^
 
-To persist Data Protection keys you must create registry hives for each application pool to store the keys. You should use the
-`Provisioning PowerShell script <https://github.com/aspnet/DataProtection/blob/dev/Provision-AutoGenKeys.ps1>`_ for each application pool you will be hosting ASP.NET Core applications under.
+Enable the **Web Server (IIS)** server role and establish role services.
 
-For web farm scenarios developers can configure their applications to use a UNC path to store the data protection key ring. By default this does not encrypt the key ring. You can deploy an x509 certificate to each machine and use that to encrypt the keyring. See the :ref:`configuration APIs <data-protection-configuring>` for more details.
+Windows Desktop Operating Systems
+-------------------------------------------------------
+Navigate to **Control Panel > Programs > Programs and Features > Turn Windows features on or off** (left side of the screen). Open the group for **Internet Information Services** and **Web Management Tools**. Check the box for **IIS Management Console**. Check the box for **World Wide Web Services**. Accept the default features for **World Wide Web Services** or customize the IIS features to suit your needs.
 
-.. include:: ./dataProtectionWarning.txt
+.. image:: pubIIS/_static/wf.png
 
-Publish from Visual Studio
------------------------------------
+Windows Server Operating Systems
+--------------------------------
+For server operating systems, use the **Add Roles and Features Wizard** via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
 
-1. Create an ASP.NET Core app using the ASP.NET Core **Web Application** template.
-2. In **Solution Explorer**, right-click the project and select **Publish**.
+.. image:: pubIIS/_static/rs.png
 
-.. image:: pubIIS/_static/p1.png
+On the **Role services** step, select the IIS role services you desire or accept the default role services provided.
 
-3. In the **Publish Web** dialog, on the **Profile** tab, select **File System**.
+.. image:: pubIIS/_static/role-services.png
 
-.. image:: pubIIS/_static/fs.png
+Proceed through the **Confirmation** step to enable the web server role and services.
 
-4. Enter a profile name. Click **Next**.
-5. On the **Connection** tab, you can change the publishing target path from the default *..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish* folder. Click **Next**.
-6. On the **Settings** tab, you can select the configuration, target runtime version, and publish options. Click **Next**.
+.. _unlock-handlers:
 
-The **Preview** tab shows you the publish path (by default, the same directory as the ".sln" solution file).
+Install the Microsoft ASP.NET Core Module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Deploy to IIS server
------------------------------------
+Install the Microsoft ASP.NET Core Module version 1.0 or higher on the server:
```

Comment:
The rc2 version of ASP.NET Core Module will not be available as a standalone .msi.

The recommended way of acquiring ANCM is installing ASP.NET Core 1.0 RC2 server bundle that contains both the .NET Redist and ANCM.

When the download link becomes live, we will want to point to that bundle.